app/src/main/java/org/sopt/sample/presentation/my_page/MyPageActivity.kt

```diff
@@ -0,0 +1,34 @@
+package org.sopt.sample.presentation.my_page
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.viewModels
+import dagger.hilt.android.AndroidEntryPoint
+import org.sopt.sample.R
+import org.sopt.sample.databinding.ActivityMyPageBinding
+import org.sopt.sample.presentation.setting.SettingActivity
+import org.sopt.sample.util.binding.BindingActivity
+
+@AndroidEntryPoint
+class MyPageActivity : BindingActivity<ActivityMyPageBinding>(R.layout.activity_my_page) {
+
+    private val viewModel by viewModels<MyPageViewModel>()
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setMyPageInfo()
+        initClickListener()
+    }
+
+    private fun setMyPageInfo() {
+        binding.tvMbti.text =
+            String.format(getString(R.string.my_page_mbti), viewModel.userMbti.value)
+    }
+
+    private fun initClickListener() {
+        binding.ivSetting.setOnClickListener {
+            Intent(this, SettingActivity::class.java).also { intent ->
+                startActivity(intent)
+            }
```

Comment:
```suggestion
            startActivity(Intent(this, SettingActivity::class.java))
```
오호라.. also군요ㅎㅎ 더 간단하게 이렇게 작성해보는 건 어떨까용?!
사실 어떤 것을 사용해도 상관은 없을 것 같은데 also에서 반환된 객체를 사용하는 것은 아니라면 let을 사용해볼 수도 있겠네요 :)

Reply:
흠냐.. 고치겠습니다

Reply:
> 사실 어떤 것을 사용해도 상관은 없을 것 같은데 also에서 반환된 객체를 사용하는 것은 아니라면 let을 사용해볼 수도 있겠네요 :)  

그런데, let도 (T)->R 블럭에서 반환된 객체를 사용하는 것이 아니기 때문에.. 흐음

Reply:
그렇긴하죠 ㅎㅎ 깔끔하게 해당 경우에는 범위지정함수 사용하지 않는게..

Reply:
헤헷.. 그래서 지웠어요..